packages/theming/atlas/src/themesource/atlas_core/native/core/widgets/dropdown.ts

```diff
@@ -92,7 +92,6 @@ export const DropDown: DropDownType = {
     },
     itemContainer: {
```

Comment:
Let's also add the `width: "100%"` here :)

Reply:
Can do, although it won't have any functional effect as the widget in client defaults `width` to `"100%". 

I can think of a benefit of doing it if `maxWidth: undefined` is added here as well; then if customers update Atlas but not Studio Pro, the issue would be resolved. 

@wesleysieses What do you think?

Reply:
Yes that's a good one. Let's add that one too.
I agree that it will not have any functional effect. But it improves the developer experience as Atlas will contain all applied styling. So a developer doesn't need a debugger to know how to apply their own custom styling

Reply:
Changed